hack/make-bundle-vars.mk

```diff
@@ -129,13 +137,17 @@ CEPH_CSI_PACKAGE_VERSION ?= 4.19.0
 # noobaa dependencies
 NOOBAA_PACKAGE_VERSION ?= 5.19.0
 
+# snapshotter-controller dependencies
+SNAPSHOT_CONTROLLER_VERSION ?= 4.20.0
```

Comment:
SNAPSHOT_CONTROLLER_VERSION to SNAPSHOT_CONTROLLER_PACKAGE_VERSION

Reply:
Done